client/rest-high-level/build.gradle

```diff
@@ -111,7 +111,7 @@ testClusters.configureEach {
   setting 'indices.lifecycle.poll_interval', '1000ms'
   setting 'indices.lifecycle.history_index_enabled', 'false'
   keystore 'xpack.security.transport.ssl.truststore.secure_password', 'testnode'
-  extraConfigFile 'roles.yml', file('roles.yml')
+  rolesFile file('roles.yml')
```

Comment:
Yes, that's proposed on the original PR that introduced `rolesFile` -> https://github.com/elastic/elasticsearch/pull/82137#issuecomment-1002841065

I haven't done it here, because it's not related to this PR (and it's a moving target). It will be easier to introduce once everything is converted over.